spirv_glsl.cpp

```diff
@@ -778,16 +772,28 @@ void CompilerGLSL::emit_header()
 		{
 			if (options.version < 310)
 				SPIRV_CROSS_THROW("At least ESSL 3.10 required for fragment shader interlock.");
-			require_extension_internal("GL_NV_fragment_shader_interlock");
+			statement("#extension GL_NV_fragment_shader_interlock : enable");
```

Comment:
Is there any reason this is not using the require_extension_internal mechanism? I'd expect this code to request GL_ARB_fragment_shader_interlock, and then the code that emits extensions will handle the ifdefs. This is similar to how other conditional extensions do it.

Reply:
can you give an example?